R/compare.R

```diff
@@ -142,11 +142,16 @@ order_proxy <- function(proxy, direction = "asc", na_value = "largest") {
   }
 
   if (is.data.frame(proxy)) {
-    args <- unname(proxy)
-    args$decreasing <- decreasing
-    args$na.last <- na.last
+    # Avoid empty data frames getting a `NULL` order
```

Comment:
I think it's more like: Work around type-instability in base::order()

Reply:
Makes sense, I'll change